### ISSUE TITLE
Create initial model to represent FIX 4.4 header that is shared by all FIX message types

### ISSUE THREAD

**Issue by gasparin93** [gasparin93/fix-engine-model#1]
- [BeginString (8)](https://www.onixs.biz/fix-dictionary/4.4/tagNum_8.html) the FIX version of the message - must be the first tag in the messageMsgSeqNum (34)
- [BodyLength (9)](https://www.onixs.biz/fix-dictionary/4.4/tagNum_9.html): message size in bytes - must be the second tag in the message
- [MsgType (35)](https://www.onixs.biz/fix-dictionary/4.4/tagNum_35.html): the message type - must be the third tag in the message
- [SenderCompID (49)](https://www.onixs.biz/fix-dictionary/4.4/tagNum_49.html): firm sending the message
- [TargetCompID (56)](https://www.onixs.biz/fix-dictionary/4.4/tagNum_56.html): firm receiving the message
- [MsgSeqNum (34)](https://www.onixs.biz/fix-dictionary/4.4/tagNum_34.html): sequence number of message 
- [SendingTime (52)](https://www.onixs.biz/fix-dictionary/4.4/tagNum_52.html): time when message was sent, always UTC

I went outside of scope in the code implementation of this ticket into developing the New Order (Single) message type. The main reason was to refine the Header as I contextualize it within a Message. 

In other words, this ticket will now encompass the initial drafts for:
- New Order (Single) model.
- Header
